### PR TITLE
Upgrade to 2.2.0, and refer to new doc site

### PR DESCRIPTION
**Pull request by chalin**
- upgrade to angular2 2.2.0
- “Angular Dart” -> AngularDart
- Replace/remove references to angular.io

### CODE REVIEW

File: common/pipes/pubspec.yaml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 name: pipes
-description: Angular Dart API Examples for Pipes
+description: AngularDart API Examples for Pipes
 version: 0.0.1
 environment:
   sdk: '>=1.19.0 <2.0.0'
 dependencies:
-  angular2: 2.0.0-beta.22
```
Comment (kwalrath) [inline common/pipes/pubspec.yaml:7]: In the future, we should constrain the angular2 version more tightly. For example:

```
  angular2: '>=2.2.0 <2.3.0'
```

We should update all of our samples & templates to match (see [web-angular/pubspec.yaml](https://github.com/google/stagehand/blob/master/templates/web-angular/pubspec.yaml) for a good example to copy).

Comment (chalin) [reply common/pipes/pubspec.yaml:7]: As pointed out in the [pub documentation for the caret syntax](https://www.dartlang.org/tools/pub/dependencies#caret-syntax), `^2.2.0` means
> the range of all versions **guaranteed to be backwards compatible** with [2.2.0]

Why is this not good enough?

Comment (kwalrath) [reply common/pipes/pubspec.yaml:7]: Because AngularDart is not adhering to that definition, unfortunately.

Comment (filiph) [reply common/pipes/pubspec.yaml:7]: Not yet, anyway. AngularDart is allowing itself to have breaking API changes in minor (x.*y*.z) releases.

Until this is fixed, we have to do the version constraining this way.

Comment (chalin) [reply common/pipes/pubspec.yaml:7]: Ok, I'll match the stagehand `pubspec.yaml`.

Comment (chalin) [reply common/pipes/pubspec.yaml:7]: Followup PR #5

**Pull request merged**